app/code/core/Mage/Adminhtml/Block/Widget/Form.php

```diff
@@ -143,6 +143,7 @@ protected function _beforeToHtml()
     {
         $this->_prepareForm();
         $this->_initFormValues();
+        Mage::dispatchEvent('init_form_values_after', ['form' => $this]);
```

Comment:
I'd like to change 2 things:
1. Event name should follow naming convention
2. The param `form` should be `$this->getForm()`, and `$this` is assigned to param `block`.

```suggestion
Mage::dispatchEvent(
    'adminhtml_block_widget_form_init_form_values_after',
    [
        'block' => $this,
        'form' => $this->getForm()
    ]
);
```